src/connectedk8s/azext_connectedk8s/_constants.py

```diff
@@ -104,6 +105,11 @@
 Error_enabling_Features = 'Error while updating agents for enabling features. Please run \"kubectl get pods -n azure-arc\" to check the pods in case of timeout error. Error: {}'
 Error_disabling_Features = 'Error while updating agents for disabling features. Please run \"kubectl get pods -n azure-arc\" to check the pods in case of timeout error. Error: {}'
 Proxy_Kubeconfig_During_Deletion_Fault_Type = 'Encountered proxy kubeconfig during deletion.'
+Custom_Registry_Disable_Auto_Upgrade_Fault_Type = 'Auto upgrade should be disabled when using custom container registry for Arc agents and extensions.'
+Custom_Registry_Agent_Version_Required_Fault_Type = 'Agent version needs to be provided when using custom container registry for Arc agents and extensions.'
+Custom_Repository_Disable_Auto_Upgrade_Fault_Type = 'Auto upgrade should be disabled when using custom repository.'
```

Comment:
I don't see this being used anywhere else in the PR. Is it being used somewhere?

Reply:
There are being used in validator.py and custom.py